source/agora/test/Flash.d

```diff
@@ -456,18 +456,20 @@ public class FlashNodeFactory (FlashListenerType = FlashListener)
     }
 
     /// Shut down all the nodes
-    public void shutdown ()
+    public override void shutdown (bool printLogs = false)
     {
+        super.shutdown();
+
         foreach (address; this.addresses)
             enforce(this.flash_registry.unregister(address.to!string));
 
         foreach (address; this.listener_addresses)
             enforce(this.listener_registry.unregister(address));
 
-        foreach (node; this.nodes)
+        foreach (node; this.flash_nodes)
             node.shutdownNode();
 
-        foreach (node; this.nodes)
+        foreach (node; this.flash_nodes)
             node.ctrl.shutdown();
```

Comment:
Does the shutdown order matter ? Should we shutdown the flash nodes before Agora ?